packages/react/src/_availableComponents.js

```diff
@@ -0,0 +1,46 @@
+export default Object.freeze([
```

Comment:
I understand why we would want an Icon registry. What's the thinking behind listing available components also? I could see myself easily forgetting to add a new, undocumented component to this list and pat myself on the back when tests pass.

Reply:
@eskfung @aneyzberg 

I've now split the tests into two, so the intention is clearer. The first test ensures that all of the components we want to export, are actually being exported. The second test ensures that all of the exported components have prop-types and docgen-info.

I also don't want to manually update this list. Although, it's difficult to generate with how the project is structured right now. I'd normally generate the list from the contents of a directory or `package.json` dependencies.